src/test/assembly/asm/hexagon-types.rs

```diff
@@ -108,7 +108,7 @@ pub unsafe fn sym_fn() {
 // CHECK: InlineAsm Start
 // CHECK: {
 // CHECK:   r{{[0-9]+}} = r0
-// CHECK:   memw(r1) = r{{[0-9]+}}
+// CHECK:   memw(r1{{(\+#0)?}}) = r{{[0-9]+}}
```

Comment:
This offset is implied in the baseline and it would appear that the code emitted may have changed slightly.

This test change is appropriate.